packages/shared/src/components/post/SquadPostContent.tsx

```diff
@@ -88,26 +86,23 @@ function SquadPostContent({
         className={classNames(
           'relative tablet:pb-0',
           className?.container,
-          isPublicSquad ? 'flex-1 flex-col tablet:flex-row' : '!pb-2',
+          'flex-1 flex-col tablet:flex-row',
         )}
         hasNavigation={hasNavigation}
       >
         <div
           className={classNames(
             'relative min-w-0 px-4 tablet:px-8',
             className?.content,
-            isPublicSquad && 'flex flex-1 flex-col',
+            'flex flex-1 flex-col',
```

Comment:
Move this in with the other just above (`relative min-w...`)

Reply:
Ah yes, I knew I wanted to do something still 😅
Thanks for the sharp eyes